app/server/static/components/label.vue

```diff
@@ -52,23 +52,16 @@
                 label.label Color
                 div.field.has-addons
                   div.control
-                    a.button(
-                      v-bind:style="{ \
-                        color: newLabel.text_color, \
-                        backgroundColor: newLabel.background_color \
-                      }"
+                    .form__field
+                        .form__input
```

Comment:
This probably fails puglint since other places in the code use explicit `div` prefixes.